datadog_checks_dev/datadog_checks/dev/tooling/commands/validate/models.py

```diff
@@ -176,7 +178,8 @@ def models(ctx, check, sync, verbose):
                 expected_model_file_lines.extend(generated_model_file_lines)
 
             # If we're re-generating a file, we should ensure we do not change the license date
-            if len(current_model_file_lines) > 0:
+            # We also want to handle the case where there is no license header
+            if len(current_model_file_lines) > 0 and LICENSE_HEADER in current_model_file_lines[0]:
```

Comment:
Is this supposed to be `LICENSE_HEADER not in current_model_file_lines` since it' handling the case where file doesn't have license header?